code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -55,12 +57,25 @@
 	icon_name = "l_arm"
 	max_damage = 50
 	min_broken_damage = 30
+	body_damage_coeff = 0.75
 	w_class = WEIGHT_CLASS_NORMAL
 	body_part = ARM_LEFT
 	parent_organ = "chest"
 	amputation_point = "left shoulder"
 	can_grasp = 1
 	convertable_children = list(/obj/item/organ/external/hand)
+	stam_heal_tick = 2
+	max_stamina_damage = 50
+
+/obj/item/organ/external/arm/set_disabled(new_disabled = TRUE)
+	..()
```

Comment:
You should default to this unless you have a good reason so that you can save some work later when you need a return value. The cost is minimal and you get a nice benefit in terms of readability as well if it's standard
```suggestion
	. = ..()
```